### PR TITLE
Feature/rnd 132 make poet great again

### PR DESCRIPTION
**Pull request by PouriaSterling**
Changes in this PR:
1. Update the runtime of Poet to nodejs8.10 and make all neccesary changes to get him up and running again.
2. Convert the use of lambda invoke to an SNS SQS interaction for improved observability
3. Update missing information in README
4. Add a new intents for locking, unlocking and checking the status of a Jenkins deployment endpoint.

### CODE REVIEW

File: src/handler.js [hunk 1]
```diff
@@ -88,21 +88,14 @@ module.exports.receptionist = (event, context, callback) => {
...
-                console.log("Invoke success: " + data.Payload);
-                context.succeed(data.Payload)
-            }
-        });
+        // publish the event to the SNS topic for the event lambda to pickup
+        console.log('Publishing event to SNS queue. ARN: ', TOPIC_ARN);
+        sns.publish({
+            Message: JSON.stringify(event),
+            TopicArn: TOPIC_ARN
+        }).promise()
```
Comment (dspasojevic) [inline src/handler.js]: Out of interest, why not use `await` here?

File: src/services/JenkinsService.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+const axios = require("axios");
```
Comment (dspasojevic) [inline src/services/JenkinsService.js:1]: Nice!

File: src/services/Utils.js [hunk 3]
```diff
@@ -38,3 +38,47 @@ module.exports.authorizedHTML = () =>
 			<p>Thanks and enjoy!</p>
 		</body>
 	</html>`;
+
+// A list of Custom Jenkin's emojis expected to be found in Slack
+// TODO Add these emojis to the repo
+const JenkinsEmojis = [
```
Comment (dspasojevic) [inline src/services/Utils.js:44]: We got the important stuff covered off 😅

**Review by dspasojevic — COMMENTED** [review]
Looks good. One questions. Otherwise, LGTM.

File: src/handler.js (hunk 1 above)
Comment (PouriaSterling) [reply src/handler.js]: Good point, I've converted it to use await.

**Review by dspasojevic — APPROVED** [review]

**Pull request merged**